### PR TITLE
Bump minimum supported `node` version.

### LINKED ISSUE #35: Consider bumping minimum supported `node` version.
If you drop unsupported `node` versions and support `node>=4`, you can drop the `string.prototype.codepointat` dependency.  You can also make things a little prettier.  For example,
```
var matchingKeys = emojiKeys.filter(function(key) {
  return key.toString().indexOf(matcher) === 0;
});
```
becomes
```
const matchingKeys = emojiKeys.filter(key => key.toString().indexOf(matcher) === 0);
```

### PR DESCRIPTION
**Pull request by wtgtybhertgeghgtwtg**
Proposed to close #35.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-  - "0.10"
+  - "4"
```
Comment (sibelius) [inline .travis.yml]: can we add more nodes?

4,5,6,7 ?

**Comment by sibelius** [comment]
@omnidan this PR looks good to me

**Comment by omnidan** [comment]
Thanks a lot @wtgtybhertgeghgtwtg and @sibelius!

I have manually merged this into the [2.0 branch](https://github.com/omnidan/node-emoji/tree/2.0), where some more refactoring will be done and 0.x support will be dropped entirely.

File: .travis.yml (hunk 1 above)
Comment (omnidan) [inline .travis.yml]: @sibelius I added node 5, 6 and 7

**Pull request closed without merging**